soepy/simulate/simulate_auxiliary.py

```diff
@@ -80,8 +82,15 @@ def pyth_simulate(
 
         # Convert to init age of child
         child_init_age = np.where(kids_init_draw == 0, -1, 0)
-        # Add column to state space
-        initial_states_in_period = np.c_[initial_states_in_period, child_init_age]
+
+        # Draw presence of partner in the first period
+        # Discuss: Best way to implement random draws by education
```

Comment:
Can we remove `Discuss`? .. througout the PR.

Reply:
Done.